pkg/cmd/cli/cli.go

```diff
@@ -63,7 +63,7 @@ func NewCommandCLI(name, fullName string) *cobra.Command {
 	out := os.Stdout
 
 	cmds.AddCommand(cmd.NewCmdLogin(f, in, out))
-	cmds.AddCommand(cmd.NewCmdProject(f, out))
+	cmds.AddCommand(cmd.NewCmdProject(fullName, f, out))
```

Comment:
While you're rebasing, can you update this to look more like the line right below?  The intent is to pass the short name of the command to get the `Use` right and the `fullName` to be the complete name without any appending.  So `"project"` and `fullName + " project"`.  

Don't forget to update the long description when you make the change.